perfeed/settings/pr_summary_prompts.toml

```diff
@@ -69,6 +72,6 @@ PR comments:
 '{{comments}}'
 ======
 
-Response (should be a valid PRSummary json, and nothing else):
-```json
+Response only json, no description before or after.
```

Comment:
I tested with ollama's llama3.1:8b, but the response would not be json. 
How's your result?

Reply:
@henry60603 , which PR did you test with? I wonder if this is due to the complexity of the PR

Reply:
this is possible. let's keep eyes on this.